airbyte-webapp/src/components/ColumnSortButton/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { FormattedMessage } from "react-intl";
+
+import styles from "./ColumnSortButton.module.scss";
+import SortIcon from "./SortIcon";
+
+interface ColumnSortButtonProps {
+  onClick: () => void;
+  formattedMessageId: string;
+  wasActive: boolean;
+  lowToLarge: boolean;
+}
+
+export const ColumnSortButton: React.FC<ColumnSortButtonProps> = ({
+  onClick,
+  formattedMessageId,
+  wasActive,
+  lowToLarge,
+}) => (
+  <button className={styles.sortButton} onClick={onClick}>
+    <FormattedMessage id={formattedMessageId} />
```

Comment:
The preference is not to pass the messageId but instead just pass the content, so one suggestion is to make this `{children}` and let the user of this component add `<FormattedMessage id="...." />` as the child.